apps/entity-service/src/file/file-upload.service.ts

```diff
@@ -98,42 +109,41 @@ export class FileUploadService {
       createdBy: this.entitiesService.userId
     };
 
-    const dbMedia = await Media.create(media as Media);
+    console.log(media);
 
-    return new MediaDto(dbMedia, {
-      url: this.mediaService.getUrl(dbMedia),
-      thumbUrl: this.mediaService.getUrl(dbMedia, "thumbnail"),
-      entityType: entity,
-      entityUuid: model.uuid
-    });
+    const dbMedia = new Media(media as Media);
+    return dbMedia.save();
```

Comment:
In my previous feedback on this, I was meaning that up above when you create `media`, you could do instead as:
```
const media = new Media();
media.collectionName = collection;
... etc

await media.save();
```

I don't think it's useful to simply replace a single line `await Media.create()` with a two line new / save.